packages/bridge-ui/src/components/form/BridgeForm.svelte

```diff
@@ -76,10 +133,20 @@
   </label>
 </div>
 
-<button
-  class="btn btn-accent"
-  on:click={async () => await bridge()}
-  disabled={btnDisabled}
->
-  {$_("home.bridge")}
-</button>
+{#if !requiresAllowance}
+  <button
+    class="btn btn-accent"
+    on:click={async () => await bridge()}
```

Comment:
```suggestion
    on:click={bridge}
```
Looks like they are identical? and the approach in suggestion is more elegant?